.github/workflows/test.yml

```diff
@@ -0,0 +1,39 @@
+name: codechecker-tests
+
+# Triggers the workflow on push or pull request events.
+on: [push, pull_request]
+
+permissions: read-all
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  analyzer:
+    name: Analysis
+    runs-on: ubuntu-24.04
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+
+      - name: Setup Bazel
+        uses: abhinavsingh/setup-bazel@v3
```

Comment:
Is it official Bazel setup?

Reply:
The repo wiki page sure claims its doing the official install steps. In terms of the _official github workflow_ setup rules, this seems to be the closest, but it seems overkill and a real hassle to make it install the proper version of bazel:
https://github.com/bazel-contrib/setup-bazel